src/components/WebexAvatar/WebexAvatar.jsx

```diff
@@ -26,16 +48,28 @@ export default function WebexAvatar({
   const type = displayStatus ? status : null;
 
   const cssClasses = webexComponentClasses('avatar', className);
+  const oldClasses = webexComponentClasses('old-avatar', className);
+
+  let AvatarComponent = renderAvatarContent(avatar, type);
+
+  if (!avatar || !displayName) {
```

Comment:
it's a bit strange to call renderAvatarContent and then cancel the result here.

I would suggest:
```js
if (!avatar && !displayName) {
  content = ??loading??;
} else if (!avatar && displayName === ' ') {
  content = ??error??;
} else {
   content = <>
     <img ... />
     {displayStatus && <div status-icon>
        ...
     </div>}
  </>;
}
```

if there is no icon name for that status, that's an error that should be reported (logged).

but if it's ok not to have an icon for a particular status, then you can check here for the icon name as well `{displayStatus && iconName && <div>...</div>}`